lib/dpl/provider/pages.rb

```diff
@@ -58,7 +58,7 @@ def github_deploy
         context.shell "git config user.email '#{@gh_email}'"
         context.shell "git config user.name '#{@gh_name}'"
         context.shell "echo '#{@gh_fqdn}' > CNAME" if @gh_fqdn
-        context.shell 'git add .'
+        context.shell 'git add . > /dev/null'
```

Comment:
Hmm. Does `git add .` show the files being added? I don't think it does, unless `-v` is given.